index.html

```diff
@@ -7,5 +7,42 @@
     <link rel="stylesheet" href="style.css" />
     <title>Portfolio</title>
   </head>
-  <body></body>
+  <body>
+    <header id="header">
+      <div id="owner_name">
+        <p>Derbal Amine</p>
+      </div>
+      <div id="header_menu">
+        <img src="images/icons8-menu-32.png" alt="header_menu" width="20px" />
+      </div>
+    </header>
+    <section id="headline">
+      <div id="introduction">
+        <p>Hey There.</p>
+        <p>I'm Amine</p>
+      </div>
+      <div id="Occupation">I'am a Software Developer</div>
+      <div id="qualification">
+        I can help you build a product , feature or website Look through some of
+        my work and experience! If you like what you see and have a project you
+        need coded, don’t hestiate to contact me.
+      </div>
+      <nav id="contact">
+        <ul id="contact_infos">
+          <li><img src="images/github_icon.png" alt="github" /></li>
+          <li>
+            <img
+              src="images/linkedin_icon_part_1.png"
+              alt="linkedin"
+              srcset=""
+            /><img src="images/linkedin_icon.png" alt="linkedin" />
+          </li>
+          <li><img src="images/contact_icon.png" alt="contact" /></li>
+          <li><img src="images/twiter_icon.png" alt="twiter" /></li>
+          <li><img src="images/mail_icon.png" alt="email" /></li>
+        </ul>
+      </nav>
```

Comment:
- [ ] Good job for using a `ul` tag for the list of icons to group them, and `li` per icon;But I recommend to use `a` tag for  each icon instead of `img` tag Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using `href`.
- [ ] Kindly I recommend you to use `div` tag for wrap the social icons instead of `nav` tag for semantic reasons.